sources/Interop/WinTrust/um/wintrust/CRYPT_PROVIDER_DATA.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/WinTrust.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct CRYPT_PROVIDER_DATA
+    {
+        [NativeTypeName("DWORD")]
+        public uint cbStruct;
+
+        [NativeTypeName("WINTRUST_DATA *")]
+        public WINTRUST_DATA* pWintrustData;
+
+        [NativeTypeName("BOOL")]
+        public int fOpenedFile;
+
+        [NativeTypeName("HWND")]
+        public HWND__* hWndParent;
```

Comment:
Remap using `HWND__*=IntPtr` for these ones.
Also `_GUID` below

Reply:
Fixed.